### ISSUE TITLE
docs for yt4.x

### ISSUE THREAD

**Issue by dnarayanan** [dnarayanan/powderday#87]
update docs to show that yt4.x no longer needs the different branch.   this means that the code needs to check for a commit hash later than X to work ...

**Comment by dnarayanan** [comment]
update: code dev. doesn't need to happen. looks like the latest versions are:

```
In [1]: import yt

In [2]: yt.__version__
Out[2]: '4.0.dev0'
```

**Comment by dnarayanan** [comment]
maybe something like this:

```if int(yt.__version__.split('.')[0]) >= 4``` to make it future proof (suggseted by sidney lower)

**Comment by dnarayanan** [comment]
fixed by 224e78a72d75f597f32b52dae00481bd583b0ffc
